### ISSUE TITLE
Allow log & Console to connect at different / configurable baudrate

### ISSUE THREAD

**Issue by marcelrv** [esphome/esp-web-tools#517]
The web flasher has a 'Log & Console' option.
As not all applications use baudrate of 11500 for the serial communication, would be great if this could be configurable rather than fixed.

e.g. we have an application that needs to communicate only at 38400 due to compatibility with other systems.
Would be nice to be able to see the log based on that baudrate rather than that it appears that it is not working.

**Comment by SalamunKawlam** [comment]
Agreed. We also have a bunch of using different baud rate for serial prints. So having an option to change the baud rate in the ui or (better yet having a baud rate parameter in the manifest file) would be a incredibly useful.

**Comment by SalamunKawlam** [comment]
@balloob can you please take a look at this?

**Comment by eried** [comment]
or maybe "autobaud" option :P

**Comment by rainerfritz** [comment]
Would be great to have this feature. Any news on it?

**Comment by ssieb** [comment]
You would be a lot better off using a real serial terminal program instead.

**Comment by rainerfritz** [comment]
I meant more to have a different baudrate for flashing and console

**Comment by ssieb** [comment]
Doesn't flashing already happen at a much faster baud rate?
But there was nothing in this issue about flashing anyway.

**Comment by rainerfritz** [comment]
Seems to me same speed. But yes I missed the topic.

**Comment by bmedici** [comment]
Hi everyone,

Any news about this feature? How could we at least change the default baudrate, or better, offer a dropdown selection to the user?
